static/styles/zulip.scss

```diff
@@ -2093,12 +2093,11 @@ div.floating_recipient {
     }
 
     &.small_square_check {
-        background-color: hsl(166, 35%, 57%);
```

Comment:
why are you changing these colors?

Reply:
just checking which suits best when the mouse hover on the buttons

Reply:
I don't think you need to change the colors at all.

Reply:
![screencapture-localhost-9991-1597742482393](https://user-images.githubusercontent.com/56171689/90495717-8f925d00-e162-11ea-8b0c-56916451e22d.png)
is it okay for I define the css indivdually for the buttons

Reply:
For this PR the buttons should look like the topic edit buttons. If the style of the topic edit buttons can be improved, that's a separate issue.

Reply:
I just made another commit I think it will be best soln for this can you check it. It only changes the color of these buttons , it will not the others